### DISCUSSION TITLE
Name already registered

### DISCUSSION THREAD

**Discussion by pir8ped** [mastodon/mastodon#20099] [Q&A]
Hi, I tried signing up with mastodonapp.uk and filled in my name and email address and got the message that my name and email are already signed up. I've never been to a mastadon site before. So, I tried just logging in, and it worked. Apparently there is an account using my name and email address, and the password suggested by Chrome worked. I'm logged in. What didn't work was getting a confirmation email.

So has someone nicked my name and email address, or is the system very broken? Any suggestions?

Thanks

**Comment by afontenot** [comment]
Note: I'm not affiliated with Mastodon.

I wonder if it's possible the sign up form glitched or you pressed the button twice by accident. In this case the account would have been created on the first press, and then the second press would error out because the account was already created. If there weren't already posts or a non-default avatar set on the account, I would assume that this is what happened and not worry about it.

**Reply by pir8ped** [reply]
Thanks for the suggestion. I don't think I pressed the button twice, but I do do a lot of button pressing, so maybe I got carried away. So I deleted my account, signed up again, and still haven't received a confirmation email. Checked spam, asked to resend etc.

**Reply by mohe2015** [reply]
Potentially the server is simply overloaded or not configured correctly. you can see in the about page if there is some information or link to a support page. this is probably something only the instance administrator can help with.
